pwiz_tools/Skyline/Util/Adduct.cs

```diff
@@ -315,7 +315,7 @@ private void ParseDescription(string input)
                                 success = int.TryParse(multMstr, out multiplierM);
                             }
                             var isotope = m.Groups["ion"].Value; // Not L10N
-                            var unlabel = isotope.Replace("'", "").Replace("\"", ""); // Not L10N
+                            var unlabel = BioMassCalc.DICT_HEAVYSYMBOL_TO_MONOSYMBOL.Aggregate(isotope, (current, kvp) => current.Replace(kvp.Key, kvp.Value));

```

Comment:
Is "isotope" a chemical formula here? I'm worried that we're just doing string replace on a chemical formula, and so "T" will match any element which starts with a "T".

You might want to use SequenceUtil.TokenizeFormula.

Reply:
Excellent point, thanks!

Reply:
On closer inspection, this isn't anything to worry about until we start supporting heavy Ti labeling.